apps/web/src/app/(pages)/(dashboard)/roles/page.tsx

```diff
@@ -22,6 +24,24 @@ export default function Roles({
     workEnvironment?: WorkEnvironmentType;
   };
 }) {
+  const [error, setError] = useState<string | undefined>(undefined);
+
+  useEffect(() => {
+    const isValidTerm = searchParams?.workTerm
```

Comment:
i'm fairly certain we don't need to check for random invalid search params, like if someone puts josh=best-pl-ever, we can safely ignore

Reply:
just a comment, not a change